### DISCUSSION TITLE
Storybook query param addon does not work properly

### DISCUSSION THREAD

**Discussion by florinpavel22** [storybookjs/storybook#17580] [General]
I tested this [query param addon](https://storybook.js.org/addons/@storybook/addon-queryparams) with the latest Storybook version and with Storybook 6.2.9. From my understanding, this addon should allow us to add/change query params in the URL, which would be useful because we're trying to set up some conditional rendering of stories based on the query params. But in both Storybook versions, the addon doesn't add anything to the URL.

Furthermore, in Storybook 6.2.9, I get this in the console.
```
Omitted potentially unsafe URL args.

More info: https://storybook.js.org/docs/react/writing-stories/args#setting-args-through-the-url
```

**Comment by shilman** [comment]
Queryparams is no longer maintained in this repo. Please see https://github.com/storybookjs/addon-queryparams
